source/base/polynomials_barycentric.cc

```diff
@@ -92,6 +92,57 @@ BarycentricPolynomials<dim>::get_fe_p_basis(const unsigned int degree)
             }
           break;
         }
+      case 3:
+        {
+          // vertices, then lines, then quads:
```

Comment:
Did you mean this?
```suggestion
          // vertices, then lines, then triangles:
```
Or "cell" as the last part?